File: labs/lab02/Combinador.java
package labs.lab02;

import java.util.Scanner;

public class Combinador {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        while (true) {
            String string1 = sc.next();
            String string2 = sc.next();
            
          
            if (string1.equals(" ") || string2.equals(" ")) {
                break;
            }

            int tam1 = string1.length();
            int tam2 = string2.length();

            if (tam1 < tam2) {
                for (int i = 0; i < tam1; i++) {
                    System.out.print(string1.charAt(i));
                    System.out.print(string2.charAt(i));
                }
                for (int i = tam1; i < tam2; i++) {
                    System.out.print(string2.charAt(i));
                }
            } else {
                for (int i = 0; i < tam2; i++) {
                    System.out.print(string1.charAt(i));
                    System.out.print(string2.charAt(i));
                }
                for (int i = tam2; i < tam1; i++) {
                    System.out.print(string1.charAt(i));
                }
            }
        }
        sc.close();
    }
}

File: labs/lab02/Combinador.c
#include <stdio.h>
#include <string.h>

int main() {
    char string1[100], string2[100];

    while (1) {
 
        scanf("%s %s", string1, string2);
        
        if (strcmp(string1, " ") == 0 || strcmp(string2, " ") == 0) {
            break;
        }

        int tam1 = strlen(string1);
        int tam2 = strlen(string2);

        if (tam1 < tam2) {
            for (int i = 0; i < tam1; i++) {
                printf("%c%c", string1[i], string2[i]);
            }
            for (int i = tam1; i < tam2; i++) {
                printf("%c", string2[i]);
            }
        } else {
            for (int i = 0; i < tam2; i++) {
                printf("%c%c", string1[i], string2[i]);
            }
            for (int i = tam2; i < tam1; i++) {
                printf("%c", string1[i]);
            }
        }
        printf("\n");
    }

    return 0;
}
